code/style.css

```diff
@@ -1,4 +1,227 @@
-html {
-  background: #0025ff;
-  color: #fff;
+
+@import url('https://fonts.googleapis.com/css2?family=Amatic+SC&family=Montserrat:wght@100&family=Open+Sans+Condensed:wght@300;700&display=swap');
+
+/* Mobile  */
+
+body {
+  box-sizing: border-box;
+  margin:0%;
+  font-family: 'Montserrat', sans-serif;
+  background-color: #f2f2f0;
 }
+  
+/* <!-- Header  -->  */
+
+header {
+  height: 130px;  
+  display: flex;  
+  align-items: center;  
+  padding: 16px;
+  color:#d0c8b6;
+ } 
+
+nav a {
+   text-decoration: none;
+   padding: 20px;
+}
+
+.polaroid {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+   width: 280x
```

Comment:
I think you can remove this line if you want.

Reply:
I tried, but it wasn´t possible. The frame dissapeard.